online-shop/src/app/components/containers/product-details/product-details.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import {
+  getProductDetail,
+  getProductsDetail,
+} from 'src/app/mocks/products.mocks';
+import { ProductDetail } from 'src/app/modules/shared/types/products.types';
+
+@Component({
+  selector: 'app-product-details',
+  templateUrl: './product-details.component.html',
+  styleUrls: ['./product-details.component.scss'],
+})
+export class ProductDetailsComponent implements OnInit {
+  productDetail: ProductDetail | undefined;
+  constructor(private route: ActivatedRoute) {}
```

Comment:
It's worth adding some spacings in between the rows, let the code "breathe"

Reply:
solved (also many blank spaces added in my code to let it breathe).